lib/rubocop/cop/ezcater/graphql_fields_naming.rb

```diff
@@ -66,7 +66,7 @@ def method_name(node)
         end
 
         def message(style)
-          format(MSG, style: style)
+          format(MSG, style: style) # Rubocop:disable Style/HashSyntax
```

Comment:
I pulled this branch down and yanked the `disable` out and Rubocop didn't bark. Could we remove this or is there something else here that I missed?

Reply:
If you try with the newest rubocop, it barks on that, so to support newer dependencies, it should probably stay.